roles/docker_storage_to_overlay2/tasks/main.yml

```diff
@@ -0,0 +1,137 @@
+---
+
+# As of 7.4, SELinux security labels are now supported
+# on the OverlayFS file system (see RHEL release notes).
+- name: Check if distribution supports overlayfs (RHEL/CentOS)
+  when: ansible_distribution in ["RedHat", "CentOS"]
+  assert:
+    that: ansible_distribution_version | version_compare("7.4", "ge")
+    msg: "The OverlayFS file system only works with SELinux in {{ ansible_distribution }} 7.4 or later."
+
+- name: Install or update required packages
+  package:
+    name: "{{ item.key }}"
+    state: latest
+  with_dict: "{{ package_requirements }}"
```

Comment:
What packages are these?

Probably should prefix them by role if they are a role variable.  This will avoid any role conflicts.
```
docker_to_storage_overlay2_package_requirements
```

Reply:
I mentioned them in the README.md:
 * docker
 * atomic
 * container-storage-setup

They all required some special backporting or extra changes beyond what's in RHEL 7.4.

:+1: on the prefix